regolith/helpers/a_expensehelper.py

```diff
@@ -129,49 +128,49 @@ def db_updater(self):
         if expense_type == 'travel':
             pdoc.update({'itemized_expenses': [
                 {
-                    'date': date,
+                    'date': date_parser.parse(date_string).date(),
```

Comment:
...then here youshould just need ``'date': begin_date``

Reply:
This is the strange behaviour I was referring to, if I use this line the value for date becomes date: *id001. For some reason, putting the date object in a list more than once results in this behaviour. The fix I had in mind was to simply make a date object for each key, which is why I use the string since it allows me to call the constructor.

Reply:
I see.  To avoid this we may have to modify the ``dump_yaml`` in fsclient to ignore aliases (see https://stackoverflow.com/questions/58091449/disabling-alias-for-yaml-file-in-python).  Why don't you try this trick on your branch and see if it works.

Reply:
Thank you! It does work.

Reply:
cool.  Please could you also include in "news" that we made this change to the yaml dumper.  This is a better fix because it will remove this problem from wherever dump is used.....